types/testing-library__dom/wait-for-element-to-be-removed.d.ts

```diff
@@ -1,5 +1,5 @@
 export function waitForElementToBeRemoved<T>(
-    callback: () => T,
+    callback: (() => T) | HTMLElement | HTMLElement[],
```

Comment:
```suggestion
    callback: (() => T) | T,
```
is how we document this as well.